Sources/SnapshotTesting/AssertSnapshot.swift

```diff
@@ -7,6 +7,7 @@ import XCTest
 /// diffTool = "ksdiff"
 /// ```
 public var diffTool: String? = nil
+public var diffToolSuffix: String = ""
```

Comment:
What about rework these two into a reusable, tool-independent message converter?
I'm suggesting this due to the following.  Although, the solution offered works fine with these 2 tools, but what about using separate switches to prefix each item (for example: `another_img_diff_tool -d 42 -i img1.png -i img2.png`? 

Something like this (at the moment I have not bothered to deprecate the old variables, but that would complete the picture):

```swift
public var diffMessageConverter: (String, String) → String? = { "\(diffTool) \($0) \($1)" }
```

Reply:
good suggestion
[8959584](https://github.com/pointfreeco/swift-snapshot-testing/pull/668/commits/8959584fcfedfd0ca93f45509e809656e564a760)